catboost/private/libs/options/enum_helpers.cpp

```diff
@@ -12,6 +12,8 @@
 #include <util/generic/map.h>
 #include <util/generic/ptr.h>
 #include <util/string/cast.h>
+#include <util/generic/ymath.h>
```

Comment:
Why it is required here?

Reply:
Fixed. It was present from an old code proposal.